fortio/Makefile

```diff
@@ -0,0 +1,22 @@
+
+deploy-fortio:
+	istioctl kube-inject --debug=false -f fortio.yaml | kubectl apply -f -
+
+cert-setup:
+	# TODO: more granular rbac roles
+	-kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
+	helm init
+	helm repo update
+	helm install stable/cert-manager
+
+ingress-setup:
+	kubectl apply -f ingress.yaml
+
+cert-issue:
+	kubectl apply -f cert.yaml
+
+# dangerous
```

Comment:
Why we still want this dangerous option here? Maybe add more comments here?

Reply:
This explanation would be useful context to add to the comment.

Reply:
Adding text